ush/sample_config_AQM/config.yaml_community_aqmna13km_hera

```diff
@@ -0,0 +1,90 @@
+metadata:
+  description: config for Online-CMAQ, AQM_NA_13km, community mode on HERA
+user:
+  RUN_ENVIR: community
+  MACHINE: hera
+  ACCOUNT: naqfc
+workflow:
+  USE_CRON_TO_RELAUNCH: true
+  CRON_RELAUNCH_INTVL_MNTS: 3
+#  EXPT_BASEDIR: /scratch2/NCEPDEV/stmp3/Chan-hoo.Jeon/expt_dirs
+  EXPT_SUBDIR: aqm_community_aqmna13
+  PREDEF_GRID_NAME: AQM_NA_13km
+  CCPP_PHYS_SUITE: FV3_GFS_v16
+  DATE_FIRST_CYCL: '2023011700'
+  DATE_LAST_CYCL: '2023011706'
+  INCR_CYCL_FREQ: 6
+  FCST_LEN_HRS: 6
+#  FCST_LEN_HRS: -1
+#  FCST_LEN_CYCL:
+#    - 6
+#    - 12
+  PREEXISTING_DIR_METHOD: rename
+  VERBOSE: true
+  COMPILER: intel
+  DIAG_TABLE_TMPL_FN: diag_table_aqm.FV3_GFS_v16
+  FIELD_TABLE_TMPL_FN: field_table_aqm.FV3_GFS_v16
+nco:
+  NET: aqm
+workflow_switches:
+  RUN_TASK_MAKE_GRID: true
+  RUN_TASK_MAKE_OROG: true
+  RUN_TASK_MAKE_SFC_CLIMO: true
+  RUN_TASK_RUN_POST: false
+task_get_extrn_ics:
+  EXTRN_MDL_NAME_ICS: FV3GFS
+  FV3GFS_FILE_FMT_ICS: netcdf
+  EXTRN_MDL_ICS_OFFSET_HRS: 0
+task_get_extrn_lbcs:
+  EXTRN_MDL_NAME_LBCS: FV3GFS
+  LBC_SPEC_INTVL_HRS: 6
+  FV3GFS_FILE_FMT_LBCS: netcdf
+  EXTRN_MDL_LBCS_OFFSET_HRS: 0
+task_run_fcst:
+  DT_ATMOS: 180
+  LAYOUT_X: 50
+  LAYOUT_Y: 34
+  BLOCKSIZE: 16
+  RESTART_INTERVAL: 6
+  WTIME_RUN_FCST: 02:00:00
+  QUILTING: true
+  PRINT_ESMF: false
+task_run_post:
+  POST_OUTPUT_DOMAIN_NAME: 793
+global:
+  DO_ENSEMBLE: false
+  NUM_ENS_MEMBERS: 2
+  HALO_BLEND: 0
+  DO_REAL_TIME: false
+cpl_aqm_parm:
+  CPL_AQM: true
+  RUN_TASK_AQM_ICS: true
+  RUN_TASK_AQM_LBCS: true
+  RUN_TASK_NEXUS_GFS_SFC: true
+  RUN_TASK_NEXUS_EMISSION: true
+  RUN_TASK_FIRE_EMISSION: true
+  RUN_TASK_POINT_SOURCE: true
+  RUN_TASK_PRE_POST_STAT: true
+  RUN_TASK_POST_STAT_O3: false
+  RUN_TASK_POST_STAT_PM25: false
+  RUN_TASK_BIAS_CORRECTION_O3: false
+  RUN_TASK_BIAS_CORRECTION_PM25: false
+  DO_AQM_CHEM_LBCS: true
+  DO_AQM_GEFS_LBCS: true
+  AQM_BIO_FILE: BEIS_RRFScmaq_C775.ncf
+  DO_AQM_DUST: true
+  AQM_DUST_FILE_PREFIX: FENGSHA_p8_10km_inputs
+  AQM_DUST_FILE_SUFFIX: .nc  
+  DO_AQM_CANOPY: false
+  AQM_CANOPY_FILE_PREFIX: gfs.t12z.geo
+  AQM_CANOPY_FILE_SUFFIX: .canopy_regrid.nc
+  AQM_FIRE_FILE_PREFIX: Hourly_Emissions_regrid_NA_13km
+  AQM_FIRE_FILE_SUFFIX: _h72.nc
+  AQM_RC_FIRE_FREQUENCY: hourly
+  DO_AQM_PRODUCT: true
+  AQM_LBCS_FILES: am4_bndy.c793.2019<MM>.v1.nc
+  AQM_GEFS_FILE_PREFIX: geaer
+  NEXUS_GRID_FN: grid_spec_793.nc
+  NUM_SPLIT_NEXUS: 3
+#  COLDSTART: false
+#  WARMSTART_CYCLE_DIR: /scratch2/NCEPDEV/naqfc/RRFS_CMAQ/restart/RRFS_CONUS_13km/2019080612
```

Comment:
Not sure if we can put a test case that runs only on Hera under WE2E but once you get the test case running on all platforms I suppose this folder should probably be moved there.

Reply:
The sample scripts of the community and nco modes for Online-CMAQ were moved to the parent directory as others. They are renamed as 'config.aqm.community.yaml' and 'config.aqm.nco.realtime.yaml', respectively.